apps/communications/contacts/test/unit/views/list/boot_test.js

```diff
@@ -0,0 +1,164 @@
+/* global LazyLoader, ListController, ListUI, MockCache */
+
+'use strict';
+
+requireApp('communications/contacts/js/param_utils.js');
+requireApp('communications/contacts/js/utilities/performance_helper.js');
+requireApp('communications/contacts/views/list/js/boot.js');
+requireApp('communications/contacts/test/unit/mock_cache.js');
+require('/shared/test/unit/load_body_html_helper.js');
+require('/shared/js/lazy_loader.js');
+
+
+suite('List - Boot', function() {
+  var stub, realCache;
+  suiteSetup(function() {
+    stub = sinon.stub(LazyLoader, 'load', function(files) {
+      return {
+        then: function(cb) {
+          cb();
+        }
+      };
+    });
+
+    realCache = window.Cache;
+    window.Cache = MockCache;
+  });
+
+  suiteTeardown(function() {
+    LazyLoader.load.restore();
+    window.Cache = realCache;
+    realCache = null;
+  });
+
+  setup(function() {
+    loadBodyHTML('/contacts/views/list/list.html');
+  });
+
+  teardown(function() {
+    document.body.innerHTML = '';
+  });
+
+  suite('DOMContentLoaded', function() {
+    setup(function() {
+      window.dispatchEvent(new CustomEvent('DOMContentLoaded'));
+    });
+
+    test(' > LazyLoader must be called once', function() {
+      // We must have 2 calls to LazyLoader
```

Comment:
nit: comment should be 'We must have 1 call to LazyLoader' as I'm understanding this test